### PR TITLE
Feat/homebrew gh action

### PR DESCRIPTION
**Pull request by morrieinmaas**
This works now @blu3beri - lemme know if you have any feedback

### CODE REVIEW

File: .github/workflows/brew-release.yml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+      - main
+
+jobs:
+  release-brew:
+    name: release brew
+    runs-on: ubuntu-latest
+    steps:
+      - name: Get Ruby file
+        id: get_ruby_file
+        run: curl -o aries-cli.rb https://raw.githubusercontent.com/animo/homebrew-aries-cli/main/Formula/aries-cli.rb
```
Comment (blu3beri) [inline .github/workflows/brew-release.yml:23]: Awesome! With this we can host the rb file on this repo :D

File: .github/workflows/brew-release.yml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+      - name: Push file to homebrew repo
+        uses: dmnemec/copy_file_to_another_repo_action@main
+        env:
+          API_TOKEN_GITHUB: ${{ secrets.HOMEBREW_GITHUB_API_TOKEN }}
+        with:
+          source_file: './aries-cli.rb'
+          destination_repo: 'animo/homebrew-aries-cli'
+          destination_folder: 'Formula'
+          user_email: 'development@animo.id'
+          user_name: 'animoNinja'
```
Comment (blu3beri) [inline .github/workflows/brew-release.yml:43]: That's not our name

Comment (morrieinmaas) [reply .github/workflows/brew-release.yml:43]: what should I use then?

File: .github/workflows/brew-release.yml [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+          sed -i -e "7s|.*|    url \"$URL_X86\"|" aries-cli.rb
+          # replace intel sha
+          sed -i -e "8s|.*|    sha256 \"$SHA_X86\"|" aries-cli.rb
+          # replace arm download url 
+          sed -i -e "12s|.*|    url \"$URL_ARM\"|" aries-cli.rb
+          # replace arm sha
+          sed -i -e "13s|.*|    sha256 \"$SHA_ARM\"|" aries-cli.rb
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Push file to homebrew repo
```
[the remark this answers is not in the document]
Comment (morrieinmaas) [reply .github/workflows/brew-release.yml:34]: yeah this now pushed to the animo/homebrew-aries-cli repo

**Comment by morrieinmaas** [comment]
alright i'll add that depends on then for only running this when new binaries are available

defo also wanna submit it to homebrew/core straight away. 

just wanted to get this working at least in a a way that's reproducible and automated and gets the job done aka lets e/o install it using brew

**Comment by morrieinmaas** [comment]
Thanks for your feedback!

File: .github/workflows/brew-release.yml (hunk 2 above)
Comment (anagoessens) [reply .github/workflows/brew-release.yml:43]: You can use: contact@animo.id

Comment (blu3beri) [reply .github/workflows/brew-release.yml:43]: or animo-bot. As its just a commit author name

**Comment by morrieinmaas** [comment]
@blu3beri as we discussed briefly last week, can we merge this as a tab and I'll do the brew/core update in a different PR?

**Pull request merged**